game-core/src/main/java/games/strategy/engine/lobby/common/LobbyConstants.java

```diff
@@ -9,6 +10,8 @@
   public static final String ADMIN_USERNAME = "Admin";
   public static final String LOBBY_CHAT = "_LOBBY_CHAT";
   public static final Version LOBBY_VERSION = new Version(1, 0, 0);
+  public static final RemoteName MODERATOR_CONTROLLER_REMOTE_NAME =
+      new RemoteName(IModeratorController.class, "games.strategy.engine.lobby.server.ModeratorController:Global");
```

Comment:
`RemoteName` is immutable, so it should be safe to share a single instance everywhere.  Not sure why the legacy code created a new instance every time.

Reply:
I realize that I am not sure what you are doing :)  But it would probably be good to test if you can boot a game, boot a bot, and check that you can test all mod tool box functions.  Just a thought great work!

Reply:
Thanks for the heads-up, @prastle.  Fortunately, due to the nature of this particular change, testing one of the moderator functions is just as good as testing them all.  So we should be good.

Reply:
@ssoloff I was able to find the original commit that added this functionality b4ed229f5132388044a6713cbd33567dbb0f4000 
No hint on why it was done this way, I suspect using a method instead of a static field doesn't force an object to stay in the heap for the lifetime of the JVM?
Not sure how the JVM handles static objects.